helm/hpcc/templates/esp.yaml

```diff
@@ -17,6 +17,7 @@ spec:
         accessDali: "yes"
         app: {{ $application }}
     spec:
+      {{- include "hpcc.placementMatchName" (dict "root" $ "pod" .name "cluster" .name "type" "esp") | indent 6 }}
```

Comment:
"cluster" doesn't really make sense in this context, perhaps should not be set.

Reply:
I think my "cluster" is "target" in your word. For example there are two item "roxie1" and "roxie2" in "roxie" type.  The same for esp such as "eclwatch", "eclservice". It is true the pod and cluster/target will be the same thing for esp.  cluster/target is mostly for roxie and thor types

Reply:
As discussed offline, we're trying to deprecate the term "cluster" which has been over used to mean many quite disparate things.
We've said we'd call the job queues that a client can target "queue"'s , I think "jobQueue" may help disambiguate it from other internal queues (at least when naming method/variables etc).

Those jobQueue targets, have various pods associated with them, e.g. an hthor fires off per workunit pods. A Thor has 2 agent pods, and fires off manager and worker pods. Eclccserver fires off compile pods.
So it _may_ make sense go group though in some sense, other than by type.

But Esp, (and Dali and other shared services), don't belong to a job queue target grouping like that.

In general, I think it would be enough if you allowed a wildcard match on pod name.
And if all pods (including pods launched by others), contained the name of jobqueue targethat launches them, i.e. instead of the current "compile-....", it was "myeclcc-compile-.."

By type could also be interesting, e.g. all thorworkers .. but if naming convention correct, you could do that with name matching only.